server.py

```diff
@@ -832,6 +832,15 @@ async def skipped_boss_relics(ctx: ContextType, save: Savefile): # JSON_FP_PROP
 
     await ctx.send(" ".join(msg))
 
+@with_savefile("bottle", "bottled")
+async def bottled_cards(ctx: ContextType, save: SaveFile):
+    """List all bottled cards."""
+    cards = save.bottled_cards
+    if len(cards) > 0:
```

Comment:
Using `len` to check for emptiness is bad practice. Just say `if cards` instead!